README.md

```diff
@@ -75,6 +77,18 @@ $query = $queryBuilder
 $result = $navitia->call($query); // Calls 'coverage/fr-idf/traffic_reports'
 ```
 
+Using single navitia instance:
+``` php
+$navitia = new Navitia($myClient, 'myDefaultCoverage');
```

Comment:
Good, it could replace my exemple on using querybuilder.
However `$navitia = new Navitia($myClient, 'myDefaultCoverage');` is confusing because we can't copy paste the exemple and make it work as is: we don't know `$myClient` comes from.
We should use the `Navitia::createFromBaseUrlAndToken` factory